assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -106,7 +110,7 @@ export default function FormInstructions() {
 		return (
 			<p>
 				{ __(
-					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below, the snippets will be inserted automatically on your site.',
+					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below. You can change these later in your settings.',
```

Comment:
Now that we're changing this string anyway, it might be a good time to break it apart. The second sentence here is (and should be) exactly the same as the one sentence below. So to make it easier for translators and avoid them having to do duplicate translations, let's use two separate translation strings here for the two sentences.

They should still remain within the same `p` though. Remember to add a space `' '` between them.

Reply:
Hi @felixarntz, 

The strings are slightly different: 

`Please select your Tag Manager account and **relevant containers** below. You can change these later in your settings.`

`Please select your Tag Manager account and container below. You can change these later in your settings`

Did you mean we can get way with the second string in both secondary AMP and normal cases? I have done it like so, but let me know if you meant something else.

Reply:
@kuasha420 Ahh, my bad, sorry about that. I didn't read them exactly, I thought the second sentence was the same.

Can you revert https://github.com/google/site-kit-wp/pull/5096/commits/f92e57fca83e84178e2c80d9483e05cefc3140d9 then?

Reply:
@felixarntz No worries, reverted. Thanks

Reply:
Thank you!